src/mngt/ocf_mngt_core.c

```diff
@@ -535,23 +533,10 @@ void ocf_mngt_cache_add_core(ocf_cache_t cache,
 	context->cache = cache;
 	context->cfg = *cfg;
 
-	name = env_vmalloc(OCF_CORE_NAME_SIZE);
```

Comment:
check at line 521 is no longer needed as well :)

Reply:
thanks for caution

Reply:
*should be changed as we discussed